core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -226,14 +243,18 @@ public static Socket connect(Properties props, String unixPathSuffix) throws IOE
     final List<String> ipTypes = listIpTypes(props.getProperty("ipTypes", DEFAULT_IP_TYPES));
     logger.info(
         String.format("Connecting to Cloud SQL instance [%s] via SSL socket.", csqlInstanceName));
-    return getInstance().createSslSocket(csqlInstanceName, ipTypes);
+    return getInstance().createSslSocket(csqlInstanceName, ipTypes, enableIamAuth);
   }
 
   /**
    * Returns data that can be used to establish Cloud SQL SSL connection.
    */
+  public static SslData getSslData(String csqlInstanceName, boolean enableIamAuth) {
+    return getInstance().getCloudSqlInstance(csqlInstanceName, enableIamAuth).getSslData();
+  }
+
   public static SslData getSslData(String csqlInstanceName) {
-    return getInstance().getCloudSqlInstance(csqlInstanceName).getSslData();
+    return getSslData(csqlInstanceName, false);
```

Comment:
I'm not sure I understand this change - can you give me some more context?

Reply:
It's just overloading `getSSLData` so that it can be called with or without an `enableIamAuth` boolean passed in. If it's not passed in it defaults to false.